src/main/kotlin/no/nav/arbeidsgiver/min_side/services/digisyfo/DigisyfoService.kt

```diff
@@ -33,12 +33,27 @@ class DigisyfoService(
             resultat.size.toString()
         ).increment()
 
-        return resultat
+        return resultat.distinctBy { it.organisasjon.organizationNumber }
     }
 
+    private fun hentForfedre(org: Organisasjon, orgs: MutableList<Organisasjon> = mutableListOf()): List<Organisasjon> {
+        if (org.parentOrganizationNumber == null) return orgs
+
+        val overenhet = eregService.hentOverenhet(org.parentOrganizationNumber!!) ?: return orgs
```

Comment:
her kan det komme nullpointer siden EregService setter parentOrganizationNumber til null
https://github.com/navikt/min-side-arbeidsgiver-api/blob/master/src/main/kotlin/no/nav/arbeidsgiver/min_side/services/ereg/EregService.kt#L93

Vi må passe på at parentOrganizationNumber ikke er null, og få den populert fra EregService. Dersom den er null kan vi ikke gå høyere opp da vi er på øverste nivå.

Reply:
her lurer vi oss selv ved at vi mocker kallet til eregservice og returnerer en hovedenhet med parentOrganizationNumber som ikke er null. Dette vil aldri skje i produksjon. Da står vi igjen med minst to muligheter:
1. flytte mockingen ned til resttemplate nivå
2. endre mockdata som returneres til å være realistisk iht slik eregservice ville gjort det.

Reply:
For loggens skyld så henter vi nå parentOrgnr for alle orgledd også. Kun juridisk enhet har parentOrgnr = null